src/file-3.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:32373/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:32373/project/issues?id=sample&pullRequest=4972&issues=AWHCSzCH1Y34ZFGmKQ41&open=AWHCSzCH1Y34ZFGmKQ41)

[ISSUE_KEY]: AWHCSzCH1Y34ZFGmKQ41